content/tutorial/tutorial.md

```diff
@@ -12,125 +12,130 @@ redirect_from:
   - "docs/tutorial-zh-CN.html"
 ---
 
-This tutorial doesn't assume any existing React knowledge.
+Deze tutorial vereist geen voorbestaande React kennis.
 
-## Before We Start the Tutorial {#before-we-start-the-tutorial}
+## Voordat we beginnen {#voordat-we-beginnen}
```

Comment:
I don't think we should translate the anchors, this way they'll still work when switching languages. I looked up an example where this also wasn't done for the Spanish translation: https://github.com/reactjs/es.reactjs.org/pull/199/files#diff-c9112233168e6093109b86f0f5b0f328R16

```suggestion
## Voordat we beginnen {#before-we-start-the-tutorial}
```

Reply:
We shouldn't no